test/aws/ec2.yml

```diff
@@ -0,0 +1,110 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  A stack for testing AWSCore on EC2.
+
+Parameters:
+  KeyName:
+    Description: The keyname associated with the instance.
+    Type: String
+    Default: ec2_keyfile
+  PublicCIUser:
+    Description: User which can assume the testing role
+    Type: String
+
+Resources:
+  EC2TravisPolicy:
+    Type: AWS::IAM::Policy
+    Properties:
+      PolicyName: EC2TravisPolicy
+      Users:
+        - !Sub ${PublicCIUser}
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Action:
+              - ec2:DescribeInstances
+              - ec2:DescribeInstanceStatus
+              - ec2:TerminateInstances
+            Resource: "*"
+          - Effect: Allow
+            Action:
+              - ec2:CreateTags
+            Resource: !Sub arn:aws:ec2:${AWS::Region}:${AWS::AccountId}:instance/*
+          - Effect: Allow
+            Action:
+              - ec2:RunInstances
```

Comment:
Here is a good example of how this should be set up: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ExamplePolicies_EC2.html#iam-example-tags-launch-template

Reply:
Thanks!